lib/instrumental/configuration.rb

```diff
@@ -0,0 +1,33 @@
+require "yaml"
+
+module Instrumental
+  class Configuration
+    FileMissing = Class.new(StandardError)
+
+    def self.load
+      if env_location
+        # this should never raise unless a bad ENV setting has been set
+        raise(FileMissing.new(env_location)) unless File.exist?(env_location)
+        return YAML.load(env_location)
+      end
+
+      if File.exist?(app_location)
+        return YAML.load_file(app_location)
+      end
+
+      YAML.load_file(gem_location)
+    end
+
+    def self.env_location
+      ENV["METRICIAN_CONFIG"]
+    end
+
+    def self.app_location
+      File.join(Dir.pwd, "config", "metrician.yaml")
```

Comment:
GitHub ate I comment I had in the previous review about config loading, so I will try to reproduce!

I'm not sure how careful everyone is about where they run their app from. I think sometimes we start rails processes from the deploy home dir instead of the app dir. So, I think we should expand the search list a little to include RAILS_ENV, RACK_ENV, and the current user's HOME dir.

I'm thinking this order:
1. ENV METRICIAN_CONFIG
2. RAILS_ENV
3. RACK_ENV
4. HOME
5. `Dir.pwd`
6. `gem_location`

Thoughts?